apps/nextjs-website/src/lib/types/product.ts

```diff
@@ -0,0 +1,9 @@
+import { Path } from '@/lib/types/path';
+
+export type Product = {
+  readonly subpaths: {
```

Comment:
What do you think to rename it in pages? [Subpaths](https://en.wiktionary.org/wiki/subpath#:~:text=(computing)%20A%20file%20or%20resource%20path%20relative%20to%20another%20path.) recall me a relative path, but as I see they are absolute. They are not relative to the product path.

```suggestion
  readonly pages: {
```

Reply:
I don't think it's a good idea ;)
as `subpaths` corresponds to a map of objects of Path type and there's also a Page type (that includes Path), I think that renaming these property to `pages` could be a source of misunderstanding